src/payment/strategies/googlepay/googlepay-adyenv3-initializer.ts

```diff
@@ -0,0 +1,111 @@
+import { round } from 'lodash';
+
+import { Checkout } from '../../../checkout';
+import PaymentMethod from '../../payment-method';
+
+import { BillingAddressFormat, GooglePaymentData, GooglePayInitializer, GooglePayPaymentDataRequestV2, TokenizePayload } from './googlepay';
+
+export default class GooglePayAdyenV3Initializer implements GooglePayInitializer {
```

Comment:
This file seems exact copy of the v2-initializer, do we need to create this file in that case?

Reply:
Yeah, it's a duplicate of the GooglePayAdyenV2Initializer. It could have a different implementation in the future; that's why I made a duplicate of it.